### ISSUE TITLE
Foundation 2.2.1 support

### ISSUE THREAD

**Issue by gabrieljt** [delight/grails-foundation#3]
Hello,

does this plugin support the latest Foundation release?

I can't get tooltips to work..

Thanks!!

-Gabriel

**Comment by delight** [comment]
Hi Gabriel,

ppl at zurb seem to have a hard time tagging there releases on github ... without tags it makes picking the right versions of the resources unnecessary  harder.. 

I issued a ticket on that
https://github.com/zurb/foundation/issues/275
but it basically got ignored on the first chance.

Don't know if I'm willing to do the diggin but pull requests are more then welcome.

Cheers,
   Kosta

**Comment by gabrieljt** [comment]
Hello delight, thanks for answering!

Good to know you're still working on this plugin, which is great by the way!

I tried to "hack" the plugin just to get the tooltips working.
In the FoundationResources.groovy, I've changed the resources url from the ui.css and the app.js, to read files from my web-app. Created a new module 'foundation-tooltips-js' and made it read the jquery,tooltips.js from the Foundation Git, also added it to the dependencies in the 'foundation-js' bundle.

In the external files, ui.css and app.js, I've added the snippets related to the tooltips from the latest Foundation files.

Well, the css is changing, but the tooltips still isnt working, and I'm aware this is definitely not a good solution.

If there is anything I can do to help you out, just contact me!

Thanks, and keep up the good work :)
